test/openzeppelin/governor/ExampleGovernor.js

```diff
@@ -107,14 +107,15 @@ describe('@OZGovernor Test Suite', function () {
     );
 
     // execute proposal
-    await governor
+    const tx = await governor
       .connect(deployer)
       .execute(
         actions.targets,
         actions.values,
         actions.calldatas,
         descriptionHash
       );
+    await tx.wait();
```

Comment:
I like these additions it helps the mirror-node has more time to update records.